config/routes.rb

```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+
+
+
+
+
+
+  get 'outings' => 'outings#index'
```

Comment:
I was under the impression that eventually, our resources would be nested. One would go to /politician/:id/outings to see all the outings for a particular politician